astroquery/hsa/tests/test_hsa_remote.py

```diff
@@ -0,0 +1,117 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import pytest
+
+import os
+
+from ..core import HSAClass
+from .test_hsa import TestHSA
+
+
+class TestHSARemote(TestHSA):
```

Comment:
Why do you need to subclass here?

Reply:
It's not needed anymore since we're not using the `dummyTapHandler` anymore.